### PR TITLE
Added QFN-32-1EP_5x5mm_P0.5mm_EP3.3x3.3mm.

### PR DESCRIPTION
**Pull request by cnieves1**
Datasheet: http://ww1.microchip.com/downloads/en/DeviceDoc/00002164B.pdf

imagen

imagen

### CODE REVIEW

**Comment by chschlue** [comment]
#422

**Comment by cnieves1** [comment]
Could you please tell me what exact changes are required by https://github.com/pointhi/kicad-footprint-generator/pull/422 ?

As I understand, it is a change in the script. 
Do I need to change any data introduced in this PR?

Thanks,

**Comment by chschlue** [comment]
see https://github.com/pointhi/kicad-footprint-generator/pull/452#issuecomment-579143412

**Comment by cnieves1** [comment]
Ok. Regenerated footprint updated at:
  https://github.com/KiCad/kicad-footprints/pull/2011

Thanks!

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml [hunk 1]
```diff
@@ -175,6 +175,66 @@ QFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
...
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://ww1.microchip.com/downloads/en/DeviceDoc/00002164B.pdf#page=68'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 5
+    tolerance: 0.1
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml]: Please use tolerances as given in the datasheet.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml [hunk 2]
```diff
@@ -175,6 +175,66 @@ QFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
...
+  body_size_y:
+    nominal: 5
+    tolerance: 0.1
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml]: Same as above.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml [hunk 3]
```diff
@@ -175,6 +175,66 @@ QFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
...
+  overall_height:
+    minimum: 0.7
+    nominal: 0.9
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml]: 0.85

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml [hunk 4]
```diff
@@ -175,6 +175,66 @@ QFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
...
+    maximum: 1
+
+  lead_width:
+    minimum: 0.18
+    maximum: 0.3
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml]: Missing nominal

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml [hunk 5]
```diff
@@ -175,6 +175,66 @@ QFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
...
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml]: Missing nominal

**Comment by chschlue** [comment]
FP merged.

**Pull request merged**